src/Driver/driver_configs.jl

```diff
@@ -110,7 +113,7 @@ end
 
 function AtmosLESConfiguration(
     name::String,
-    N::Int,
+    (polyorder_horz, polyorder_vert)::NTuple{2, Int},
```

Comment:
If you left this as `N` or maybe `N::Union{Int, NTuple{2, Int})` and then add after the `where` the following
```julia
(polyorder_horz, polyorder_vert) = isa(N, Int) ? (N, N) : N
```
then you can support both single `N` and `(polyorder_horz, polyorder_vert)`

Reply:
Thank you so much for the suggestion! I just changed it. Let me know if it looks good now.

Reply:
Should we do this for all the configs?

Reply:
Yup, absolutely. I was just making sure I got your suggestion right first. I just added for all configs